doc/examples.xml

```diff
@@ -309,3 +309,46 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="HaarGraph">
+<ManSection>
+  <Oper Name="HaarGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is a positive integer then this operation returns
+    the <E>Haar graph</E> <M>H(<A>n</A>)</M>. <P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/HaarGraph.html</URL>:
+    <P/>
+
+    <Q>A Haar graph H(<A>n</A>) is a bipartite regular vertex-transitive graph indexed by a 
+    positive integer and obtained by a simple binary encoding of cyclically adjacent 
+    vertices. Haar graphs may be connected or disconnected. </Q><P/>
+
+    The number of vertices in a given haar graph is equal to the number of digits 
+    required to represent the indexing integer <A>n</A> in binary. The <M>i</M>th vertex in the 
+    first column is connected to the <M>j</M>th vertex in the second column if the 
+    binary representation of <A>n</A> has a 1 in position (<M>j-i</M> modulo 
+    <M>m</M>) + 1 from the left, where <M>m</M> is the number of digits in the binary 
+    representation.<P/>
```

Comment:
```suggestion
    The number of vertices in the Haar graph <M>H(<A>n</A>)</M> is
    equal to twice <M>m</M>, where <M>m</M> is the number of digits
    required to represent <A>n</A> in binary.
    These vertices are arranged into bicomponents
    <C>[1..m]</C> and <C>[m+1..2*m]</C>.
    Vertices <M>i</M> and <M>j</M> in different bicomponents are
    adjacent by a symmetric pair of edges if and only if
    the binary representation of <A>n</A> has a 1 in position
    (<M>j-i</M> modulo <M>m</M>) + 1 from the left. <P/>
```

Reply:
Ah yeah, sorry, I accidentally wrote the number of vertices is equal to m, when it's actually twice that... The new explanation looks good to me, thanks.